test/tests.yml

```diff
@@ -183,21 +135,21 @@
   initVersion: v1.27.11
   upgradedVersion: v1.28.7
   infrastructures:
-    - name: aws_centos_stable
-    - name: aws_default_stable
-    - name: aws_flatcar_stable
-    - name: aws_rhel_stable
-    - name: aws_rockylinux_stable
+    - name: azure_centos_stable
+    - name: azure_default_stable
+    - name: azure_flatcar_stable
+    - name: azure_rhel_stable
+    - name: azure_rockylinux_stable
 
 - scenario: upgrade_containerd
   initVersion: v1.28.7
   upgradedVersion: v1.29.2
   infrastructures:
-    - name: aws_centos_stable
-    - name: aws_default_stable
-    - name: aws_flatcar_stable
-    - name: aws_rhel_stable
-    - name: aws_rockylinux_stable
+    - name: azure_centos_stable
+    - name: azure_default_stable
+    - name: azure_flatcar_stable
+    - name: azure_rhel_stable
+    - name: azure_rockylinux_stable
```

Comment:
This scenario is not possible because there are no in-tree CCMs in v1.29.

Reply:
Dropped!